web-common/src/features/dashboards/hooks/useCellInspector.ts

```diff
@@ -0,0 +1,101 @@
+import { cellInspectorStore } from "../stores/cellInspectorStore";
+
+export function useCellInspector() {
+  let hoverTimeout: number | null = null;
+  let lastHoveredValue: string | null = null;
+  let isOpen = false;
```

Comment:
Would be good for this to be tracked in a central place, perhaps in `cellInspectorStore` itself. (Which it already seems to be, so this `isOpen` might just be redundant)

Same with `lastHoveredValue`